UPGRADING.md

```diff
@@ -5,6 +5,8 @@ Upgrading Grape
 
 Grape now supports, but doesn't require Rack 1.6.0. If you encounter an issue with parsing requests larger than 128KB, explictly require Rack 1.6.0 in your Gemfile.
 
+Key route_info is excluded from params. See [#879](https://github.com/intridea/grape/pull/879) for more information.
```

Comment:
This looks confusing, the description is above a Gemfile example that relates to the note above. Make it two sections, something like:

```
### Added Rack 1.6.0 Support

...

### Removed route_info

```

Also please properly backquote things like `params`.